google-cloud-tools-plugin/jps-plugin/src/com/google/cloud/tools/intellij/appengine/jps/model/impl/JpsAppEngineModuleExtensionImpl.java

```diff
@@ -74,7 +77,7 @@ public void applyChanges(@NotNull JpsAppEngineModuleExtensionImpl modified) {
   @Override
   public String getToolsApiJarPath() {
     return FileUtil.toSystemDependentName(
-        JpsArtifactPathUtil.appendToPath(getSdkHomePath(), LIB_APPENGINE_TOOLS_API_JAR));
+        JpsArtifactPathUtil.appendToPath(getSdkHomePath(), LIB_APPENGINE_TOOLS_API_JAR.toString()));
```

Comment:
Yea definitely. This is inline with @joaoandremartins's suggestions to mark todos to offload things to the common lib where it makes sense.